CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
-# Contributing to MagTape
+	# Contributing to MagTape
```

Comment:
nitpick(blocking): I don't think this extra whitespace is needed

Reply:
Thanks, @phenixblue for reviewing my first submission.
I have added another commit to my PR after removing the leading whitespace from the title. Kindly review.